support-frontend/assets/helpers/paymentIntegrations/readerRevenueApis.js

```diff
@@ -40,27 +40,37 @@ export type StripePaymentMethod = 'StripeCheckout' | 'StripeApplePay' | 'StripeP
 export type StripePaymentRequestButtonMethod = 'none' | StripePaymentMethod;
 
 type RegularContribution = {|
+  productType: 'Contribution',
   amount: number,
   currency: string,
   billingPeriod: BillingPeriod,
 |};
 
 export type DigitalSubscription = {|
+  productType: 'DigitalPack',
```

Comment:
I would suggest using the types from `SubscriptionProduct` for these
```suggestion
  productType: DigitalPack,
```

Reply:
will do.
The only thing I wondered is I wouldn't want someone to change that and not realise it would affect the client->server payload.  But I will leave that to your judegment!